easybuild/easyconfigs/p/pftoolsV3/pftoolsV3-3.2.12-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'CMakeMake'
+
+name = 'pftoolsV3'
+version = '3.2.12'
+
+homepage = 'https://github.com/sib-swiss/pftools3'
+description = """A suite of tools to build and search generalized profiles (protein and DNA)."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/sib-swiss/pftools3/archive/refs/tags/']
+checksums = ['1de33d6444015942afa690d54a103ad5a3d68f416fad63f247c12fa3f71a550c']  # pftools3-3.2.12.tar.gz
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+    ('make', '4.4.1'),
+    ('pkg-config', '0.29.2'),
```

Comment:
does this really need `pkg-config`, or can we use `pkgconf`?

Reply:
I have no idea: I was simply updating the toolchain here.  Are pkg-config and pkgconf interchangeable?  If so, I guess I'd need pkgconf/1.9.3-GCCcore-12.2.0, for toolchain consistency?  (I see there's also a package called 'pkgconfig' :-(  .)

Reply:
`pkgconf` is preferred and (generally) a drop-in replacement that is more actively maintained, which is why we prefer it. I would suggest trying to switch it out, and if it doesn't build with `pkgconf` we can always revert to `pkg-config` again